test/lit/passes/inlining-const-args.wat

```diff
@@ -0,0 +1,88 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; With `-O3`, we always inline calls to functions that just call other
+;; functions with "trivial" arguments.
```

Comment:
This comment could maybe focus on constants, since the name of the file is about constants, and the test only has constants and not local. gets?